### PR TITLE
make Makefile more resilient

### PR DESCRIPTION
**Pull request by Charliekenney23**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-include .env
+-include .env
```
Comment (Charliekenney23) [inline Makefile]: Should not exit for missing `.env` file. Not everyone will be running integration tests

File: Makefile [hunk 2]
```diff
@@ -25,7 +25,12 @@ vet:
 	go vet ./...
 
 lint:
+ifndef $(shell command -v $(GOLANGCILINT) -v dot 2> /dev/null)
```
Comment (Charliekenney23) [inline Makefile]: Warn user about missing golangci-lint installation, but don't exit.

Comment (sarahelizgray) [reply Makefile]: Why not exit? The linting step seems useful, especially for outside contributors.

Comment (Charliekenney23) [reply Makefile]: My hope was that this warning would provide an easier way to call to action actually installing golangci-lint on the users machine nstead of:
```
make: golangci-lint: No such file or directory
```
But I see where you're getting at. Would you prefer that we log the installation instructions and then exit?

**Pull request merged**